### PR TITLE
Clean up OneByte tests

### PR DESCRIPTION
**Pull request by EvanHahn**
This test-only change:

- Tests various permutations of `perByte()`
- Uses [`toBeCloseTo()`][0] (see #98)

[0]: https://jestjs.io/docs/expect#tobeclosetonumber-numdigits

### CODE REVIEW

**Comment by EvanHahn** [comment]
These may already be tested in `co2.test.js`, so this may be a bad PR.

File: src/1byte.test.js [hunk 1]
```diff
@@ -2,16 +2,26 @@
...
-      expect(instance.KWH_PER_BYTE_FOR_NETWORK.toFixed(12)).toBe(expected_val);
+      expect(instance.perByte(0)).toBe(0);
+      expect(instance.perByte(0.99)).toBe(0);
+      expect(instance.perByte(-1)).toBe(0);
+    });
+
+    it("returns a result for grey energy", () => {
+      const instance = new OneByte();
+
+      expect(instance.perByte(1000)).toBeCloseTo(0.00029, 5);
```
Comment (fershad) [inline src/1byte.test.js:18]: We normally use 1 million (1000000) bytes as a test value, since it guarantees we'll have results to play with.

**Comment by fershad** [comment]
@EvanHahn thanks for this. It would actually be nice to reduce the number of tests in the `co2.test.js` file, so happy to explore moving more to the OneByte & Sustainable Web Design tests into their own files.

Using [toBeCloseTo()](https://jestjs.io/docs/expect#tobeclosetonumber-numdigits) looks good, and will definitely make our test cases more flexible as grid intensity data changes.

Do you want to use the PR to add more tests to the `1byte.test.js` file. We can tackle the OneByte tests first, and then get on to the Sustainable Web Design tests in another PR.

File: src/1byte.test.js (hunk 1 above)
Comment (EvanHahn) [reply src/1byte.test.js]: Done in commit eb7d08f184cf4a670f8fb730efb580529b25e32b.

**Comment by EvanHahn** [comment]
> It would actually be nice to reduce the number of tests in the `co2.test.js` file, so happy to explore moving more to the OneByte & Sustainable Web Design tests into their own files.

I removed two redundant tests from `co2.test.js` in 0e6b4811e24cfa49cd29da90ebf894ca8111b649.

> Do you want to use the PR to add more tests to the `1byte.test.js` file. We can tackle the OneByte tests first, and then get on to the Sustainable Web Design tests in another PR.

It looks like the `OneByte` class is fully tested by this PR. What other tests would you like to see?

Again, feel free to message me on the CAT Slack or email <me@evanhahn.com>.

**Comment by fershad** [comment]
@EvanHahn thank you for this, looks good to me.

**Pull request merged**